README.rst

```diff
@@ -1,5 +1,6 @@
 Python Client for Cloud AutoML API
 ==================================
+**AutoML API Python Client is now available in the new Vertex AI.** Migrate your AutoML resources to Vertex AI to get the latest features like data labeling, AutoML-powered training, and endpoint management. `Learn More`_.
```

Comment:
It looks weirdly out of style to put this note above the release/pypi/version badges. If you'd like this as a "top banner" sort of style, consider using [notes](https://sublime-and-sphinx-guide.readthedocs.io/en/latest/notes_warnings.html).

Reply:
Looks like I got the correct RST syntax for the `note` feature, but it's not rendering the way it's supposed to on my machine. Not sure what I did wrong here.

Reply:
Hmm, my bad. It seems that GH has not been supporting rendering some of the reStructuredText directives. I think it's fine to leave it without the note structure. Feel free to explore any other style that works!